src/main/java/core/basesyntax/service/operations/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.operations.impl;
```

Comment:
```suggestion
package core.basesyntax.service.strategy.handler;
```
Same for other handlers